src/app/shared/components/combobox/style.scss

```diff
@@ -16,3 +16,18 @@ input {
   outline: none;
   padding: 16px 0 16px 16px;
 }
+
+mat-select {
```

Comment:
Doesn't it happen anywhere else that we have close icon inside mat-select? I'm asking as this one will fix it only for combobox component.

Reply:
Good point. I'll add a global class.